[Text Analytics] Regenerating the SDK using the most recent and fixed version of swagger v3.0 (#10295)
Azure/azure-sdk-for-js: sdk/textanalytics/ai-text-analytics/CHANGELOG.md

## 1.0.1 (Unreleased)

> - The `text` property of the `SentenceSentiment` interface is no longer optional because the service always returns it. This interface is used exclusively as an output type so this change does not break existing code.

Reviewer: A slight update to communicate that nothing changed on the service side or in the actual data that is returned

```suggestion
- The `text` property of the `SentenceSentiment` interface is no longer marked as optional because the service has always returned it. This interface is used exclusively as an output type so this change does not break existing code.
```

> - The `text` property of the `SentenceSentiment` interface is no longer optional because the service always returns it. This interface is implemented by output types, so this change does not break existing code that calls the library APIs.

Reviewer 2: ```suggestion
- The `text` property of the `SentenceSentiment` interface is no longer optional because the service always returns it. This interface is used exclusively as an output type so this change does not break existing code.
```

## 1.0.1 (Unreleased)

> - The `text` property of the `SentenceSentiment` interface is no longer optional because the service always returns it.

Reviewer: It would be good to mention that this interface is used only in the output types so that it does not alarm the users.

Author: Done.
